picard/formats/id3.py

```diff
@@ -330,6 +350,18 @@ def _load(self, filename):
                 if frame.desc:
                     name += ':%s' % frame.desc
                 metadata.add(name, frame.text)
+            elif frameid == 'SYLT' and frame.type == 1:
+                if frame.format == 0:
+                    raise NotImplementedError("SYLT format 0 is not supported")
```

Comment:
What is the user experience when they load a file with format == 0? Is an exception the right response or would a debug log warning message be more appropriate?

What if the format is something other than 0 or 2?

Reply:
Thanks, I forgot about that. Now a warning is logged and the tag is not added to the metadata when the format is different than 2, which is "using milliseconds as unit".